Doc/whatsnew/3.11.rst

```diff
@@ -1173,6 +1173,61 @@ Deprecated
   (Contributed by Serhiy Storchaka in :gh:`91760`.)
 
 
+Pending Removal in Python 3.12
+==============================
+
+The following APIs have been deprecated in earlier Python releases, and are now
+pending removal in Python 3.12.
+
+Python API::
```

Comment:
```suggestion
Python API:
```

Reply:
Yeah, I just noticed (see ed590cc646ca05a5ded359dbc304ccba596cbafc). Thanks for the heads-up anyway!